plugins/ngrok/provisioner.go

```diff
@@ -3,19 +3,23 @@ package ngrok
 import (
 	"context"
 	"fmt"
+	"log"
 	"os"
+	"os/exec"
 	"path/filepath"
 	"runtime"
 	"strings"
 
 	"github.com/1Password/shell-plugins/sdk"
 	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
 	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+	"github.com/hashicorp/go-version"
```

Comment:
There is also semver functionality in the stdlib: https://pkg.go.dev/golang.org/x/mod/semver#Compare. What are the differences? Could/should we use that instead?